libbeat/docs/shared-configuring.asciidoc

```diff
@@ -1,24 +1,22 @@
 //Added conditional coding to support Beats that don't offer all of these install options
 
-ifeval::["{beatname_lc}"!="auditbeat"]
-
-To configure {beatname_uc}, you edit the configuration file. For rpm and deb,
-you'll find the configuration file at +/etc/{beatname_lc}/{beatname_lc}.yml+. Under
-Docker, it's located at +/usr/share/{beatname_lc}/{beatname_lc}.yml+. For mac and win,
-look in the archive that you just extracted. There’s also a full example
-configuration file called +{beatname_lc}.reference.yml+ that shows all non-deprecated
-options.
+To configure {beatname_uc}, you edit the configuration file. 
 
+ifeval::["{beatname_lc}"!="auditbeat"]
+For rpm and deb, you'll find the configuration file at +/etc/{beatname_lc}/{beatname_lc}.yml+. 
+For mac and win, look in the archive that you just extracted.
+Under Docker, it's located at +/usr/share/{beatname_lc}/{beatname_lc}.yml+.
```

Comment:
If I get this right the only difference for auditbeat is that it does not include the docker part? Could we have only an `not auditbeat if clause` for the docker part?

Reply:
I tried that, but then it renders the docker part in a new paragraph, which I think looks weird.